cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/set_asset_claimer.rs

```diff
@@ -0,0 +1,217 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Tests related to claiming assets trapped during XCM execution.
+
+use crate::imports::*;
+use emulated_integration_tests_common::{
+	accounts::{ALICE, BOB},
+	impls::AccountId32,
+};
+
+use crate::imports::bhw_xcm_config::{LocationToAccountId, UniversalLocation};
+use frame_support::{assert_ok, sp_runtime::traits::Dispatchable};
+use xcm_executor::traits::ConvertLocation;
+
+#[test]
+fn test_set_asset_claimer_within_a_chain() {
+	let (alice_account, alice_location) = account_and_location(ALICE);
+	let (bob_account, bob_location) = account_and_location(BOB);
+
+	let amount_to_send = 16_000_000_000_000;
```

Comment:
```suggestion
	let trap_amount = 16_000_000_000_000;
```
(since we're never sending this, test traps then claims it)

Reply:
renamed